src/Compilers/CSharp/Portable/Symbols/ConstantValueUtils.cs

```diff
@@ -155,5 +157,21 @@ internal static string UnescapeInterpolatedStringLiteral(string s)
 
             return value;
         }
+
+        private sealed class CheckConstantInterpolatedStringValidity : BoundTreeWalkerWithStackGuardWithoutRecursionOnTheLeftOfBinaryOperator
+        {
+            readonly internal DiagnosticBag diagnostics;
```

Comment:
Visibility modifiers should always come first.

Reply:
Fixed, thanks!